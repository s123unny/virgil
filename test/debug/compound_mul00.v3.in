b ../core/compound_mul00.v3 5
run
bt
info l
c
q
