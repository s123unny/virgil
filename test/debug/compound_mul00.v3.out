breakpoint|../core/compound_mul00.v3 5|0
stopOnBreakpoint
bt|main|../core/compound_mul00.v3|5
variable|0|a|-1|int|false
variable|1|x|3|int|false
end
result|3
